examples/no_emit/BUILD.bazel

```diff
@@ -9,12 +10,46 @@ write_file(
     ],
 )
 
-# Shows how to run `tsc` producing no outputs at all.
+write_file(
+    name = "gen_js",
+    out = "b.js",
+    content = [
+        "export const b = 43",
+    ],
+)
+
+# Shows how to run `tsc` producing no outputs at all via no_emit=True.
 # It will use a validation action to type-check the file:
```

Comment:
No longer a validation action after #690 right?

Reply:
Correct, I'll reword a few of these